api/src/main/java/org/openmrs/module/fhir2/providers/util/FhirProviderUtils.java

```diff
@@ -51,6 +61,15 @@ public static OperationOutcome buildDelete(DomainResource resource) {
 		
 	}
 	
+	@Delete
+	public OperationOutcome deleteTask(@IdParam IdType id) {
+		Task task = taskService.delete(id.getIdPart());
+		if (task == null) {
+			throw new ResourceNotFoundException("Could not find task resource with id " + id.getIdPart() + "to delete");
+		}
+		return FhirProviderUtils.buildDelete(task);
```

Comment:
This should go to R3/4 Task Providers. Utils are common methods that can be used by no other services. I am not sure why you thought about adding it here.
look at this PR it is similar to what you work on 

https://github.com/openmrs/openmrs-module-fhir2/pull/211

Reply:
I had already updated this PR with changes [here](https://github.com/openmrs/openmrs-module-fhir2/commit/2b678a313056508d7cd7bbd73880ce73e64ea956). I guess @sherrif10 force-pushed with the wrong changes/commit. He was only supposed to add tests.

Reply:
You may be right however those changes need tests like as you said before

Reply:
@corneliouzbett  did you recognise some failing tests